testgrid/specs/full.yaml

```diff
@@ -154,18 +152,16 @@
       version: 1.2.0
     ekco:
       version: 0.6.0
-- name: k8s1184
+- name: k8s121x_containerd
   installerSpec:
     kubernetes:
-      version: 1.18.4
-    docker:
-      version: 19.03.10
+      version: 1.21.x
+    containerd:
+      version: 1.4.6
     weave:
       version: 2.6.5
     contour:
       version: 1.0.1
-    rook:
-      version: 1.0.4
```

Comment:
Correct that Rook 1.0.4 doesn't work with k8s 1.21, but you still need a CSI provisioner. Either upgrade the version of rook to `latest` or switch to `longhorn`. I trust your decision on which one needs more coverage.

Reply:
`rook` latest version is `1.0.4` . Switching to `longhorn`